LICENSE

```diff
@@ -1,6 +1,6 @@
 ISC License
 
-Copyright (c) 2018, Cam Sloan <cam@sloan.ca>
+Copyright (c) 2018, ecobee Inc. <developer@ecobee.com>
```

Comment:
👍 Thanks!
Should the date be updated as well?

Reply:
Done! and I added the rest of the changes.